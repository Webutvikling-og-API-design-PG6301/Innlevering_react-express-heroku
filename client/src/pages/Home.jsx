import React from "react";
import { Link } from "react-router-dom";
const Home = () => {
  return (
    <div>
      <h1>Home page</h1>

      <Link to="/question">
        <button>New quiz</button>
      </Link>
      <Link to="/answer/*">
        <button>Show anwers</button>
      </Link>
    </div>
  );
};

export default Home;
