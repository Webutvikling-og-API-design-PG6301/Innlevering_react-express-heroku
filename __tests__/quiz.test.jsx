import React from "react";
import ReactDOM from "react-dom";
import Test from "../client/src/pages/Test";
import Home from "../client/src/pages/Home";

describe("test", () => {
  it("should test test", () => {
    const element = document.createElement("div");
    ReactDOM.render(<Test />, element);
    expect(element.innerHTML).toMatchSnapshot();
  });
});
