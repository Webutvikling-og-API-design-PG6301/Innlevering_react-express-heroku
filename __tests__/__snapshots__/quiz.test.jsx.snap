// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test should test test 1`] = `"<div>Test</div>"`;
